lib/aws-record/record/attributes.rb

```diff
@@ -26,6 +26,19 @@ def initialize
         @data = {}
       end
 
+      # @private
```

Comment:
There wasn't an existing one (I think the one you are referring to is for ClassMethods), but I've updated the PR to use the alternate `private` syntax.